.travis.yml

```diff
@@ -36,6 +36,7 @@ before_script:
 - while ! nc -q 1 127.0.0.1 64526 </dev/null; do sleep 2; done
 - curl http://127.0.0.1:64526/get?show_env=1
 - make
+- make lint
```

Comment:
maybe do the lint before `make` to fail quickly? Seems more likely someone will forget to lint before submitting that forget to build.